.travis.yml

```diff
@@ -8,6 +8,9 @@ sudo: required
 dist: trusty
 
 language: go
+os:
+- linux
```

Comment:
I guess this is enabling x86 linux as well as ppc64 - I think that is probably the right thing, so we can sanity check that the Travis builds are working on both (to try and distinguish between architecture specific issues and generic issues).

Reply:
@grahamwhaley : Updated my commit message accordingly.